tests/std/include/range_algorithm_support.hpp

```diff
@@ -899,7 +899,7 @@ constexpr void test_in_write() {
     with_input_ranges<with_writable_iterators<Instantiator, Element2>, Element1>::call();
 }
 
-template <class Instantiator, class Element1 = int, class Element2 = int>
+template <class Instantiator, class Element1 = const int, class Element2 = int>

```

Comment:
Should all default element types used for input iterators / input ranges be changed consistently?

Reply:
That is a great point. My preference would be to separate it from a feature change like this.

Doing it in a separate PR greatly simplifies the review process

Reply:
> My preference would be to separate it from a feature change like this.

Works for me.

Reply:
I should do this in #959 and make it another omnibus test machinery update.

Reply:
FYI, I changed my mind, and decided to simply remove all the default template arguments for element types in #959.

Reply:
> I suggest your revert this conflicting change.

Oh, and pass `const int` and `int` explicitly in the `copy_n` test.

Reply:
Done